CMakeLists.txt

```diff
@@ -182,6 +182,13 @@ TARGET_INCLUDE_DIRECTORIES(${PROJECT_NAME}
   $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/include>
   $<INSTALL_INTERFACE:include>)
 
+IF(APPLE)
```

Comment:
Why did you change here?

Reply:
I'm just reapplying this: https://github.com/stack-of-tasks/eigenpy/commit/8cd9fd9a5d667b267ffbf9918a92237d120fbe49

If this is useless, I can remove it.